kebechet/managers/update/update.py

```diff
@@ -154,18 +155,32 @@ def _get_all_packages_versions() -> dict:
             raise DependencyManagementError(f"Failed to load Pipfile.lock file: {str(exc)}") from exc
 
         result = {}
-        for package_name, package_info in pipfile_lock_content['default'].items():
-            result[package_name.lower()] = {
-                'dev': False,
-                'version': package_info['version'][len('=='):]
-            }
-
-        for package_name, package_info in pipfile_lock_content['develop'].items():
-            result[package_name.lower()] = {
-                'dev': False,
-                'version': package_info['version'][len('=='):]
-            }
+        try:
+            for package_name, package_info in pipfile_lock_content['default'].items():
+                result[package_name.lower()] = {
+                    'dev': False,
+                    'version': package_info['version'][len('=='):]
+                }
 
+            for package_name, package_info in pipfile_lock_content['develop'].items():
+                result[package_name.lower()] = {
+                    'dev': False,
+                    'version': package_info['version'][len('=='):]
+                }
+        except KeyError as exc:
+            _LOGGER.info("Key Errror encountered, which is mostly due to version key.")
+            pip_url = construct_raw_file_url(self.service_url, self.slug, "Pipfile", self.service_type)
+            piplock_url = construct_raw_file_url(self.service_url, self.slug, "Pipfile.lock", self.service_type)
+            self.sm.open_issue_if_not_exist(
+                _ISSUE_UNSUPPORTED_APP,
+                lambda: ISSUE_UNSUPPORTED_APP.format(
+                    sha=self.sha,
+                    package=package_name,
+                    pip_url=pip_url,
+                    piplock_url=piplock_url,
+                    environment_details=self.get_environment_details()
+                )
+            )
         return result
```

Comment:
You can close this issue otherwise so bot manages the issue on its own.

Reply:
Close the issue, just after creating it?

Reply:
You can close it if the error does not occur anymore using `close_issue_if_exists` method on source management.